File: articles/epistemology-evidence-analytical-psychology/fortran/epistemic_score.f90
! Epistemology and Evidence in Analytical Psychology
! Fortran example: simple synthetic credibility score

program epistemic_score_demo
  implicit none

  real :: empirical_support, hermeneutic_coherence, clinical_utility
  real :: phenomenological_adequacy, contextual_specificity
  real :: methodological_explicitness, ambiguity_inflation
  real :: score

  empirical_support = 0.42
  hermeneutic_coherence = 0.84
  clinical_utility = 0.78
  phenomenological_adequacy = 0.81
  contextual_specificity = 0.68
  methodological_explicitness = 0.72
  ambiguity_inflation = 0.28

  score = 0.52 * empirical_support + 0.58 * hermeneutic_coherence + &
          0.62 * clinical_utility + 0.56 * phenomenological_adequacy + &
          0.44 * contextual_specificity + 0.60 * methodological_explicitness - &
          0.72 * ambiguity_inflation

  print *, "Synthetic credibility score:", score
end program epistemic_score_demo

File: articles/epistemology-evidence-analytical-psychology/c/epistemic_score.c
/*
Epistemology and Evidence in Analytical Psychology
C example: simple synthetic credibility score

Synthetic educational demonstration only.
Not for diagnosis, assessment, screening, surveillance, or individual evaluation.
*/

#include <stdio.h>

double credibility_score(
    double empirical_support,
    double hermeneutic_coherence,
    double clinical_utility,
    double phenomenological_adequacy,
    double contextual_specificity,
    double methodological_explicitness,
    double ambiguity_inflation
) {
    return 0.52 * empirical_support
         + 0.58 * hermeneutic_coherence
         + 0.62 * clinical_utility
         + 0.56 * phenomenological_adequacy
         + 0.44 * contextual_specificity
         + 0.60 * methodological_explicitness
         - 0.72 * ambiguity_inflation;
}

int main(void) {
    double score = credibility_score(0.42, 0.84, 0.78, 0.81, 0.68, 0.72, 0.28);
    printf("Synthetic credibility score: %.3f\n", score);
    return 0;
}
